### PR TITLE
docs(aio): structural directives. no path in navigation menu. fix

### PR DESCRIPTION
**Pull request by Yerkon**
## PR Checklist
Please check if your PR fulfills the following requirements:

- [ ] The commit message follows our guidelines: https://github.com/angular/angular/blob/master/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

```
[ ] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Build related changes
[ ] CI related changes
[x] Documentation content changes
[ ] angular.io application / infrastructure changes
[ ] Other... Please describe:
```

## What is the current behavior?

Issue Number: #21848 

## What is the new behavior?
Structural directives page link added to navigation menu

## Does this PR introduce a breaking change?
```
[ ] Yes
[x] No
```

### CODE REVIEW

**Review by kapunahelewong — APPROVED** [review]
This is a really great catch. LGTM. You may want to edit the commit message to be something like "docs(aio): put structural directives back in the nav". The structure of the part after the colon should start with the verb. "Fix" would be fine, too. Thank you so much @Yerkon!

**Review by gkalpak — APPROVED** [review]

**Comment by Yerkon** [comment]
Thanks @kapunahelewong. I will update commit message to **'docs(aio): put structural directives back in the nav'**

File: aio/content/navigation.json [hunk 1]
```diff
@@ -170,6 +170,11 @@
               "title": "Attribute Directives",
               "tooltip": "Attribute directives attach behavior to elements."
             },
+            {
+              "url":"guide/structural-directives",
+              "title": "Structural Directives",
+              "tooltip": "Structural directives manipulate the layout of the page"
```
Comment (gkalpak) [inline aio/content/navigation.json]: Can you add `.` at the end for consistency?

Comment (Yerkon) [reply aio/content/navigation.json]: 😄

**Comment by Yerkon** [comment]
@gkalpak, how long PR staying opened? There are 300 PRs. Why they are still opeded at least old?

File: aio/content/navigation.json [hunk 2]
```diff
@@ -170,6 +170,11 @@
               "title": "Attribute Directives",
               "tooltip": "Attribute directives attach behavior to elements."
             },
+            {
+              "url":"guide/structural-directives",
```
Comment (gkalpak) [inline aio/content/navigation.json]: Can you put a space after `:` for consistency?

**Comment by gkalpak** [comment]
@Yerkon, PRs remain open until they are in a mergeable state (all CIs green, cla signed, approved by team member, review comments addressed) and marked for merge. It is not practical for us to periodically check all open PRs to find out if everything had been addressed. Once you have addressed all comments, you can post a comment saying the PR is ready for us to take another look.

Once everything is fine, we mark the PR for merge (which then usually takes a few hours).

**Comment by Yerkon** [comment]
All clear, thank you!

**Comment by Yerkon** [comment]
@gkalpak, Done

**Comment by gkalpak** [comment]
I've squashed the commits for you :wink:
Waiting for CIs to be green and will mark for merge. Thx for fixing this, @Yerkon 👍

**Review by gkalpak — APPROVED** [review]

**Comment by Yerkon** [comment]
What could be better than helping to make an Angular better :sunglasses:

**Pull request closed without merging**